charts/all-in-one/templates/validator.yaml

```diff
@@ -283,6 +283,12 @@ spec:
         {{- toYaml $.Values.validator.tolerations | nindent 8 }}
       {{- end }}
       dnsPolicy: ClusterFirst
+      imagePullSecrets:
+      - name: acr-regcred
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      securityContext: {}
+      terminationGracePeriodSeconds: 30
```

Comment:
is this still necessary?

Reply:
hmm i'm not sure since that portion wasn't part of this PR(I just rearranged the original setup). I'll compare it with our remote-headless template and make adjustments.

Reply:
https://github.com/planetarium/9c-infra/pull/2290/commits/ad1d076208051acf92f085683c4b732a77cb3e77

I've removed these two configurations. PTAL